src/helpers/_module-template.scss

```diff
@@ -0,0 +1,48 @@
+/*
+ * _module-template.scss
+ *
+ * This is the basic layout of a shared module and might be handy when creating
+ * a new one.
+ */
+
+/* _module-template config */
+@import '../helpers/module-dependencies';
```

Comment:
If we put this import in `style.scss`, do we have to import this file in every module SCSS file too?